### PR TITLE
Types of Inheritance

### PR DESCRIPTION
**Pull request by cbabhulkar**
Kindly review

### CODE REVIEW

File: InheritanceTypes/multilevelinheritance.cpp [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		getline(cin,name);
+	}
+
+	void show(){
+		cout<<"Student Id="<<id<<endl;
+		cout<<"Student Name="<<name<<endl;
+	}
+};
+
+class person:public student{
```
Comment (neildhale) [inline InheritanceTypes/multilevelinheritance.cpp:31]: Person ===> class name

File: InheritanceTypes/multipleinheritance.cpp [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	}
+
+	void display(){
+		cout<<"Staff Id="<<id<<endl;
+		cout<<"Staff Name="<<Name<<endl;
+	}
+};
+
+
+class Institute:public Staff,public Student{
```
Comment (neildhale) [inline InheritanceTypes/multipleinheritance.cpp:59]: comment how order of constructor and destructor called !!!!

File: InheritanceTypes/singlelevelinheritance.cpp [hunk 3]
```diff
@@ -0,0 +1,67 @@
+#include<iostream>
+using namespace std;
+
+class student{
+	int id;
```
Comment (neildhale) [inline InheritanceTypes/singlelevelinheritance.cpp:5]: I know if there is no any specifier then it is private but it is good if we mention access specifier

**Review by neildhale — CHANGES_REQUESTED** [review]
Please address review comments.

File: InheritanceTypes/multipleinheritance.cpp [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	public:
+
+	Staff(){
+		cout<<"Staff Derived1 object created"<<endl;
+	}
+
+	~Staff(){
+		cout<<"Staff Derived1 Object Destroyed"<<endl;
+	}
+	void Entry(){
```
Comment (neildhale) [inline InheritanceTypes/multipleinheritance.cpp:46]: here function name start with capital letter and  Staff::void display() start with small letter. please decide one approach for function name defination. I would suggest function name should start with small letter. if the name is to bit the you can add capital letter e.g. getStaffEntry().  display() or displayStaffDetail().

Comment (cbabhulkar) [reply InheritanceTypes/multipleinheritance.cpp:46]: Done

**Review by neildhale — APPROVED** [review]

**Pull request merged**